pkg/cmd/runner/purge.go

```diff
@@ -12,6 +19,53 @@ var purgeCmd = &cobra.Command{
 	Short: "Purges the runner",
 	Args:  cobra.NoArgs,
 	RunE: func(cmd *cobra.Command, args []string) error {
+		cfg, err := runner.GetConfig()
+		if err != nil {
+			return err
+		}
+
+		var serverStoppedCheck bool
```

Comment:
Should be called `runnerStoppedCheck`

Reply:
Also, you need to run the health check before the prompt. There's no need for the prompt if the runner is not running.